src/main/java/com/google/sps/utils/Game.java

```diff
@@ -1,18 +1,17 @@
 package com.google.sps.utils;
 import com.google.appengine.api.datastore.Entity;
 import java.lang.IllegalArgumentException;
-import java.util.List;
 import java.util.ArrayList;
-import com.google.appengine.api.datastore.DatastoreService;
-import com.google.appengine.api.datastore.DatastoreServiceFactory;
 
 public final class Game {
-    DatastoreService datastore;
+    public final static String game_kind = "Game";
```

Comment:
Please give constants UPPERCASE_NAMES. [Constants are meant to be yelled](https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names)